dashboard/app/helpers/plc/courses_helper.rb

```diff
@@ -1,5 +1,5 @@
 module Plc::CoursesHelper
   def options_for_user_enrollment_courses
-    Plc::Course.all.pluck(:name, :id).sort
+    Plc::Course.where.not(name: 'All Of The PLC Things').pluck(:name, :id).sort
```

Comment:
I didn't want All Of The Things showing up in the dropdown for new enrollments because I didn't want anyone accidentally enrolling in it.